<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrokerStaticAddGUI</class>
 <widget class="QWidget" name="BrokerStaticAddGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>166</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BrokerStaticAddGUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>In addition to searching for scopes on mDNS, brokers can also be added here based on their static IPv4 or IPv6 addresses, in addition to their scopes.

Please use one of the following formats for the address:

    d.d.d.d                     for an IPv4 address;
    x:x:x:x:x:x:x:x       for an IPv6 address.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="scopeLabel">
       <property name="text">
        <string>Scope:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scopeEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="ipAddrLabel">
       <property name="lineWidth">
        <number>200</number>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipEdit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addBrokerButton">
     <property name="text">
      <string>Add Broker</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
